i18n/management/commands/gather_i18n_strings.py

```diff
@@ -25,3 +26,11 @@ def handle(self, *args, **options):
                 outpath = os.path.abspath(os.path.join(source_dir, model.__name__ + ".json"))
                 with open(outpath, 'w') as outfile:
                     json.dump(strings, outfile, indent=4, sort_keys=True)
+
+        # Gather strings for translation in html files.
+        # For some reason makemessages won't create the locale/ directory but will create
+        # the subdirectories
+        subprocess.call(["mkdir", "locale/"])
+        subprocess.call(["python", "manage.py", "makemessages", "-l", "en"])
+        # Django.po isn't a very useful file name
+        subprocess.call(["mv", "locale/en/LC_MESSAGES/django.po", "locale/en/LC_MESSAGES/miscellaneous.po"])
```

Comment:
I'm not convinced `miscellaneous` is all that much more useful than `django`. I think I'd prefer to stick with the default here, since it's what django expects

Reply:
Removed. We can add it back if we think of a better name.